rotkehlchen/chain/ethereum/decoding/decoder.py

```diff
@@ -522,6 +564,30 @@ def _maybe_enrich_transfers(  # pylint: disable=no-self-use
 
         return None
 
+    def _enrich_protocol_tranfers(  # pylint: disable=no-self-use
+            self,
+            token: EthereumToken,
+            tx_log: EthereumTxReceiptLog,
+            transaction: EthereumTransaction,
+            event: HistoryBaseEntry,
+            action_items: List[ActionItem],
+    ) -> None:
+        """
+        Decode special transfers made by contract execution for example at the moment
+        of depositing assets or withdrawing.
+        It assumes that the event being decoded has been already filtered and is a
+        transfer.
+        """
+        if maybe_enrich_pickle_transfers(
```

Comment:
```suggestion
        if maybe_enrich_pickle_transfers(
```
This if needs to be done only when you add more. There is no reason for this if check when there is only 1 function. Please remove.